services/agents/registry.go

```diff
@@ -768,6 +768,109 @@ func (r *Registry) StartMongoDBExplainAction(ctx context.Context, id, pmmAgentID
 	return nil
 }
 
+// StartMySQLQueryShowAction starts mySQL show query action on pmm-agent.
+func (r *Registry) StartMySQLQueryShowAction(ctx context.Context, id, pmmAgentID, dsn, query string) error {
+	aRequest := &agentpb.StartActionRequest{
+		ActionId: id,
+		Params: &agentpb.StartActionRequest_MysqlQueryShowParams{
+			MysqlQueryShowParams: &agentpb.StartActionRequest_MySQLQueryShowParams{
+				Dsn:   dsn,
+				Query: query,
+			},
+		},
+	}
+
+	agent, err := r.get(pmmAgentID)
+	if err != nil {
+		return err
+	}
+
+	agent.channel.SendRequest(aRequest)
+	return nil
+}
+
+// StartMySQLQuerySelectAction starts mySQL select query action on pmm-agent.
+func (r *Registry) StartMySQLQuerySelectAction(ctx context.Context, id, pmmAgentID, dsn, query string) error {
+	aRequest := &agentpb.StartActionRequest{
+		ActionId: id,
+		Params: &agentpb.StartActionRequest_MysqlQuerySelectParams{
+			MysqlQuerySelectParams: &agentpb.StartActionRequest_MySQLQuerySelectParams{
+				Dsn:   dsn,
+				Query: query,
+			},
+		},
+	}
+
+	agent, err := r.get(pmmAgentID)
+	if err != nil {
+		return err
+	}
+
+	agent.channel.SendRequest(aRequest)
+	return nil
+}
+
+// StartPostgreSQLQueryShowAction starts postgreSQL show query action on pmm-agent.
```

Comment:
PostgreSQL, not postgreSQL. The same thing below.

Reply:
Fixed